Src/Sarif.PatternMatcher.Sdk/ValidationResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk
+{
+    public class ValidationResult
+    {
+        public static readonly IEnumerable<ValidationResult> NoMatch = new[]
+        {
+            new ValidationResult
+            {
+                Fingerprint = default,
+                ResultLevelKind = default,
```

Comment:
this is redundant, you don't ever need to explicitly set a property to its default value;

Reply:
fixed.